docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,125 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* Add `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24604`)
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance of reading from new Delta Lake tables by compressing files
+  with `ZSTD` by default, instead of the previous `SNAPPY`.  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+* Add support for views. ({issue}`24242`)
+* Support generating sequences. ({issue}`24590`)
+* {{breaking}} Replace predicate pushdown with `min`, `max`, and `options`
+  column properties. ({issue}`24147`)
+
+## Google Sheets connector
+
+## Hive connector
+
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+* Fix reading from tables when using the Alluxio file system. ({issue}`23815`)
+
+## Hudi connector
+
+## Iceberg connector
+
+* Add `$entries` metadata table. ({issue}`24172`)
+* Add `$all_entries` metadata table. ({issue}`24543`)
+* Add `system.iceberg_tables` table function to allow listing only Iceberg tables. ({issue}`24469`)
+* Allow configuring the `parquet_bloom_filter_columns` table property. ({issue}`24573`)
+* Improve performance when listing columns. ({issue}`23909`)
+* Remove the oldest tracked version metadata files when
```

Comment:
Is this a bug fix? How can we better describe the user impact @ebyhr ? 

From quick glance it seems a clean up work that has performance benefits..